waku/v2/waku_discv5.nim

```diff
@@ -89,6 +126,15 @@ proc new*(T: type WakuDiscoveryV5,
         )
         builder.withWakuCapabilities(flags)
         builder.withMultiaddrs(multiaddrs)
+
+        if relayShardsRes.isOk():
+          let op = relayShardsRes.value
+          if op.isSome():
+            let shards = relayShardsRes.value.get()
+            let res = builder.withWakuRelaySharding(shards)
```

Comment:
I don't necessarily think it's necessary to define `shards` as a separate variable here.
Something like:

```suggestion
            let res = builder.withWakuRelaySharding(op.get())
```

If you want to be more explicit you could rename `op` to e.g. `shardsOp`